docs/src/main/paradox/project/migration-guides.md

```diff
@@ -8,31 +8,36 @@ please [migrate to Akka 2.6](https://doc.akka.io/docs/akka/current/project/migra
 
 ## Migration to Apache Pekko
 
-This is just stub documentation. It will be improved.
+These migration notes are designed for users migrating from Akka 2.6 to Pekko 1.0 and assume a basic level of Akka experience. Please feel free to submit an issue or a patch if you feel like the notes can be improved.
 
 * for Pekko jar dependencies, the groupId is "org.apache.pekko" instead of "com.typesafe.akka"
 * the jar names start with "pekko" instead of "akka" - e.g. pekko-actor_2.13.jar instead of akka-actor_2.13.jar 
 * Alpakka equivalent is "pekko-connectors" - e.g. pekko-connectors-kafka_2.13.jar instead of alpakka-kafka_2.13.jar
 * Pekko packages start with "org.apache.pekko" instead of "akka" - e.g. `import org.apache.pekko.actor` instead of `import akka.actor`
 * Where class names have "Akka" in the name, the Pekko ones have "Pekko" - e.g. PekkoException instead of AkkaException
-* Configs in `application.conf` use "pekko" prefix instead of "akka"
+* Config names use "pekko" prefix instead of "akka", e.g. `pekko.actor.provider` instead of `akka.actor.provider`
 * We have changed the default ports used by the pekko-remote module.
 * With @ref:[Classic Remoting](../remoting.md), Akka defaults to 2552, while Pekko defaults to 7355.
 * With @ref:[Artery Remoting](../remoting-artery.md), Akka defaults to 25520, while Pekko defaults to 17355.
+
+### Dependency Changes
 * The Scala 2.13/Scala 3 version of Pekko no longer includes [scala-java8-compat](https://github.com/scala/scala-java8-compat)
```

Comment:
The `version` in "Scala 2.13/Scala 3 version" should be plural

Reply:
added https://github.com/apache/incubator-pekko/pull/397/commits/42373969f416b148537699f4580e09cc3dfe713d